lib/tracer.js

```diff
@@ -3,16 +3,18 @@
 // We don't want to require strong-agent, because anything it requires will
 // become untraceable. Still, we need to use its idea of the appName, and access
 // lrtime from the addon, so require the agent internals.
-var addon = require('strong-agent/lib/addon');
+// var addon = require('strong-agent/lib/addon');
 var assert = require('assert');
-var configure = require('strong-agent/lib/config').configure;
+// var configure = require('strong-agent/lib/config').configure;
 
 exports = module.exports = tracer;
 exports.start = start;
 
 var TRACER = null;
 
-var lrtime = addon ? addon.lrtime : undefined;
+// var lrtime = addon ? addon.lrtime : undefined;
+// Fix me with appropriate lrtime value
+var lrtime = 20000;
```

Comment:
this is just a random value to get this working. Apparently this value comes from C++ libraries since this comes from addon.js which is pointing to those libraries.